src/prettytable/prettytable.py

```diff
@@ -1667,7 +1667,7 @@ def add_rows(self, rows: Sequence[RowType], *, divider: bool = False) -> None:
         for row in rows[:-1]:
             self.add_row(row)
 
-        if rows:
+        if len(rows) > 0:
```

Comment:
What do you think about this, for some symmetry with the first loop?
```suggestion
        for row in rows[-1:]:
```

Reply:
`rows[-1:]` actually means "an array which contains the last element of `rows`". So `for row in rows[-1:]` means "loop over the last element of `rows`". If you do that, then this whole block should change to:

```python
for row in rows[:-1]:
    self.add_row(row)

for row in rows[-1:]:
    self.add_row(row, divider=divider)
```

Even though it's symmetrical, to me it is weird to loop over "an array with the last element, which might be empty if the array was empty". It makes the whole if-condition _implicit_, which I don't think is a good thing.